corehq/apps/userreports/tests/test_pillow.py

```diff
@@ -189,6 +189,53 @@ def test_process_doc_from_sql(self, datetime_mock):
 
         CaseAccessorSQL.hard_delete_cases(case.domain, [case.case_id])
 
+    @patch('corehq.apps.userreports.specs.datetime')
+    @override_settings(TESTS_SHOULD_USE_SQL_BACKEND=True)
+    def test_process_filter_no_longer_pass(self, datetime_mock):
+        datetime_mock.utcnow.return_value = self.fake_time_now
+        sample_doc, expected_indicators = get_sample_doc_and_indicators(self.fake_time_now)
+
+        since = self.pillow.get_change_feed().get_latest_offsets()
+
+        # save case to DB - should also publish to kafka
+        case = _save_sql_case(sample_doc)
```

Comment:
is it necessary to save the doc here - can't you just do `self.pillow.process_change(doc_to_change(sample_doc))`

That would also save having to cleanup afterwards